clang/test/CAS/mccas-replay-test.cpp

```diff
@@ -0,0 +1,19 @@
+// RUN: rm -rf %t && mkdir -p %t
+// RUN: export LLVM_CACHE_CAS_PATH=%t/cas && %clang-cache \
+// RUN:   %clang -target x86_64-apple-macos11 -c -Xclang -fcas-backend -Rcompile-job-cache %s -o %t/tmp.o 2>&1 | FileCheck %s -check-prefix=CACHE-MISS
```

Comment:
Requires x86 target?

Reply:
Are you saying that I replace this with a `REQUIRES` instead? Or are you saying I shouldn't set a target option to the compiler at all?

Reply:
Generating object file for x86 requires x86 target to be built. It doesn't matter what target you pick, you need a `REQUIRES` that matches that target.